content/en/docs/new-articles/_index.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Adding a New Article"
+linkTitle: "New Articles"
+weight: 20
+---
+
+{{% pageinfo %}}
+To contribute new articles, you need to have a [GitHub account](https://github.com)!
+{{% /pageinfo %}}
+
+Thanks for your interest in submitting documentation articles!
+This article shows you how to create your own documentation how-tos and get them published on the iXsystems [documentation website](docs.ixsystems.com).
```

Comment:
```suggestion
This article shows you how to create your own documentation how-to 's and get them published on the iXsystems [documentation website](docs.ixsystems.com).
```

I read "how-tos" funny. Would it help to put that `'s` there?

Reply:
I think it's awkward either way, but the `'s` seems a little more so. Probably best to use a different word entirely.